src/Evolution/Systems/NewtonianEuler/System.hpp

```diff
@@ -5,15 +5,62 @@
 
 #include <cstddef>
 
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Evolution/Conservative/ConservativeDuDt.hpp"
+#include "Evolution/Systems/NewtonianEuler/Characteristics.hpp"
+#include "Evolution/Systems/NewtonianEuler/ConservativeFromPrimitive.hpp"
+#include "Evolution/Systems/NewtonianEuler/Fluxes.hpp"
+#include "Evolution/Systems/NewtonianEuler/PrimitiveFromConservative.hpp"
+#include "Evolution/Systems/NewtonianEuler/Tags.hpp"
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace Tags {
+template <class>
+class Variables;
+}  // namespace Tags
+
 /// \ingroup EvolutionSystemsGroup
 /// \brief Items related to evolving the Newtonian Euler system
 namespace NewtonianEuler {
 
-template <size_t Dim>
+template <size_t Dim, typename EquationOfStateType>
 struct System {
   static constexpr bool is_in_flux_conservative_form = true;
   static constexpr bool has_primitive_and_conservative_vars = true;
   static constexpr size_t volume_dim = Dim;
+  static constexpr size_t thermodynamic_dim =
+      EquationOfStateType::thermodynamic_dim;
+
+  // Compute item for pressure not currently implemented in SpECTRE,
```

Comment:
Why isn't a compute item added? Why is this different from GRMHD?

Reply:
If I'm not wrong, one would like to add the pressure as a generic compute item, where the function to compute it is retrieved from the equation of state. What is done in this system *is* essentially the same that was done in GRMHD. 

If we implement the compute item, we might also want to do it for all hydro systems at once. Definitely something for a different PR.

Reply:
I think the reason we can't currently have a compute item is we need PDALs to get const global cache things into the DataBox. Future problem it is :)